package.json

```diff
@@ -23,20 +23,21 @@
   },
   "homepage": "https://github.com/EspeoBlockchain/gardener-smart-contracts",
   "devDependencies": {
-    "chai": "^4.1.2",
-    "chai-as-promised": "^7.1.1",
-    "coveralls": "^3.0.2",
-    "dotenv": "^6.1.0",
-    "eslint": "^5.4.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.14.0",
-    "lodash": "^4.17.10",
-    "openzeppelin-solidity": "1.12.0",
-    "solidity-coverage": "^0.5.11",
-    "solium": "^1.1.8",
-    "truffle": "4.1.14",
-    "truffle-flattener": "^1.2.10",
-    "truffle-hdwallet-provider": "^1.0.0-web3one.1"
+    "chai": "4.2.0",
+    "chai-as-promised": "7.1.1",
+    "coveralls": "3.0.3",
+    "eslint": "5.16.0",
+    "eslint-config-airbnb-base": "13.1.0",
+    "eslint-plugin-import": "2.17.3",
+    "lodash": "4.17.11",
+    "openzeppelin-solidity": "2.3.0",
+    "solidity-coverage": "0.6.0-beta.5",
+    "ethlint": "1.2.4",
+    "truffle": "5.0.20",
+    "truffle-flattener": "1.3.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "dotenv": "8.0.0",
```

Comment:
Why are these in `dependencies`?

Reply:
We use them in truffle config which is detected by IDE as a non-dev file (which is particularly true  😄 ).